unstructured_ingest/v2/processes/connectors/kafka/cloud.py

```diff
@@ -27,7 +27,7 @@
 
 class CloudKafkaAccessConfig(KafkaAccessConfig):
     api_key: Optional[SecretStr] = Field(
-        description="Kafka API key to connect at the server", alias="kafka_api_key", default=None
+        description="Kafka API key to connect at the server", default=None
```

Comment:
The alias `kafka_api_key` was causing the api_key to be None, even though it was set directly:

>       assert connection_config.access_config.get_secret_value().api_key.get_secret_value() == "..."
E       AttributeError: 'NoneType' object has no attribute 'get_secret_value'

Reply:
Ouch, now see the CI complains about "ValueError: the following field name were reused, all must be unique: api_key" (https://github.com/Unstructured-IO/unstructured-ingest/actions/runs/11956245021/job/33332171861?pr=257). I think about renaming it to `kafka_api_key` (not alias, direct rename).